getting-started/002-search.md

```diff
@@ -1,19 +1,17 @@
-`SEARCH`, or Looking for Places
-=======
+# Mapzen Search: Finding places
 
-Geospatial search, frequently referred to as **geocoding** is the process of matching an address to its corresponding geographic coordinates. There's nothing inherent in the words we use to describe an address that conveys its location at some coordinates on earth, i.e. *[lat,lon]*. Making the leap from text to coordinates is an intricate and challenging process. Lucky for you, we've done all the hard work and made it accessible via a really simple and free web service.
+Geospatial search, frequently referred to as geocoding, is the process of matching an address to its corresponding geographic coordinates. There's nothing inherent in the words we use to describe an address that conveys its position at some coordinates on earth, such as a latitude,longitude. Making the leap from text to coordinates is an intricate and challenging process. Lucky for you, Mapzen has done all the hard work and made it accessible though a free web service.
```

Comment:
i'm finding this a bit of a mouthful for the beginning of the piece, maybe we can break this paragraph up to make it more easily digestible?

eg:

> Geospatial search, commonly referred to as geocoding, is the process of matching an address to its corresponding geographic coordinates.

> There's nothing inherent in the language we use to describe a physical address that allows us to convert that human readable sentence in to a format that a computer can understand, such as latitude & longitude.

> Making the leap from text to coordinates is an intricate and challenging process. Lucky for you, Mapzen has done all the hard work and made it accessible though a free web service.